README.md

```diff
@@ -378,6 +378,16 @@ To check that, you are able to run simple curl command:
 curl --header "REMOTE_USER: Daniel" -XGET http://localhost:9876/api/0/whoami
 ```
 
+## Using API
+
+Below there are describe few examples, how to use API in Monocle:
+
+```
+curl -XGET "http://localhost:9876/api/0/query/last_changes?index=zuul&repository=.*&gte=2020-12-24&from=0&size=100&state=OPEN&selfMerged=false&ec_same_date=true"
+curl -XGET "http://localhost:9876/api/0/projects?index=openstack"
+curl -XGET "http://localhost:9876/api/0/query/changes?index=monocle&project_definition=test"
+```
+
```

Comment:
I think this should be handled in another PR dedicated to improve the documentation for the API usage.